app/code/Magento/Widget/Block/Adminhtml/Widget/Options.php

```diff
@@ -163,6 +163,8 @@ protected function _addField($parameter)
             }
         }
 
+        $data['value'] = html_entity_decode( $data['value'] );
```

Comment:
The `\Magento\Framework\Escaper::escapeHtmlAttr` function doesn't give the desired result. `html_entity_decode` is used by Magento core elsewhere. See https://github.com/magento/magento2/search?utf8=✓&q=html_entity_decode&type=Code. I have removed the extra spaces.